### DISCUSSION TITLE
12V external output power connector

### DISCUSSION THREAD

**Discussion by yoshida6298** [HumanSupportRobot/QandA#90] [Q&A] [answered]
I would like to use the 12V external output power connector on the HSRB access panel. What terminals are compatible with this connector? Also how many amps can this connector be used with?
![IMG_5650](https://github.com/HumanSupportRobot/QandA/assets/29244885/3093a4b5-7ead-4a66-8dca-e4f0fdb951c6)

**Comment by yoshimi-iyoda** [comment, marked as answer]
It can be used up to 0.5[A].
For terminal specifications, please refer to this information.
https://www.satoparts.co.jp/products/search/SL/SL-6000-V

**Reply by yoshida6298** [reply]
Thank you for your response. I will refer to it.
